OrbitGl/GraphTrack.cpp

```diff
@@ -86,26 +72,73 @@ void GraphTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
     if (it != values_.begin()) --it;
     uint64_t previous_time = it->first;
     double last_normalized_value = (it->second - min_) * inv_value_range_;
+    float kDotRadius = 2.5;
+    float base_y = pos_[1] - size_[1];
+    time_graph_->GetBatcher().AddCircle(
+        Vec2(time_graph_->GetWorldFromTick(previous_time),
+             base_y + static_cast<float>(last_normalized_value) * size_[1]),
+        kDotRadius, GlCanvas::kZValueText, kLineColor);
     for (++it; it != values_.end(); ++it) {
       if (previous_time > max_ns) break;
       uint64_t time = it->first;
       double normalized_value = (it->second - min_) * inv_value_range_;
-      float base_y = pos_[1] - size_[1];
       float x0 = time_graph_->GetWorldFromTick(previous_time);
       float x1 = time_graph_->GetWorldFromTick(time);
       float y0 = base_y + static_cast<float>(last_normalized_value) * size_[1];
       float y1 = base_y + static_cast<float>(normalized_value) * size_[1];
-      time_graph_->GetBatcher().AddLine(Vec2(x0, y0), Vec2(x1, y0), GlCanvas::kZValueText,
-                                        kLineColor);
-      time_graph_->GetBatcher().AddLine(Vec2(x1, y0), Vec2(x1, y1), GlCanvas::kZValueText,
+      // TODO: Draw only the dots when it isn't a continuous variable?
```

Comment:
Could we keep both implementations in prevision of having some UI to toggle between both?

Reply:
What do you mean? Comment these lines?

Reply:
I would remove the comment.